src/aiowaqi/converters.py

```diff
@@ -0,0 +1,154 @@
+"""Conversion functions for AQI values."""
+
+from collections.abc import Sequence
+from dataclasses import dataclass
+
+
+@dataclass
+class PollutantInfo:
+    """Data structure for pollutant information."""
+
+    unit: str
+    name: str
+    range: Sequence[int | None]
+    scale: Sequence[float]
+
+
+# AQI breakpoints and scales for different pollutants
+POLLUTANT_DATA = {
+    "pm25": PollutantInfo(
+        unit="μg/m³",
+        name="PM2.5",
+        range=[0, 50, 100, 150, 200, 300, 400, 500],
+        scale=[0, 12, 35.5, 55.5, 150.5, 250.5, 350.5, 500.5],
+    ),
+    "pm10": PollutantInfo(
+        unit="μg/m³",
+        name="PM10",
+        range=[0, 50, 100, 150, 200, 300, 400, 500],
+        scale=[0, 55, 155, 255, 355, 425, 505, 605],
+    ),
+    "o3_1h": PollutantInfo(
+        unit="ppb",
+        name="O3 (1 hour average)",
+        range=[None, 100, 150, 200, 300, 400, 500],
+        scale=[0, 0.125, 0.165, 0.205, 0.405, 0.505, 0.605],
+    ),
+    "o3_8h": PollutantInfo(
+        unit="ppb",
+        name="O3 (8 hours average)",
+        range=[0, 50, 100, 150, 200, 300, None],
+        scale=[0, 0.06, 0.076, 0.096, 0.116, 0.375],
+    ),
+    "no2": PollutantInfo(
+        unit="ppb",
+        name="NO2",
+        range=[0, 50, 100, 150, 200, 300, 400, 500],
+        scale=[0, 0.054, 0.101, 0.361, 0.65, 1.25, 1.65, 2.049],
+    ),
+    "co": PollutantInfo(
+        unit="ppm",
+        name="CO",
+        range=[0, 50, 100, 150, 200, 300, 400, 500],
+        scale=[0, 4.5, 9.5, 12.5, 15.5, 30.5, 40.5, 50.5],
+    ),
```

Comment:
Why do we put the unit and name here?

Reply:
Oh, I think my initial goal was to also return the info in the conversion function, but I forgot to do it lol. Do you think it makes sense to also return this ? I mean the unit at least can be pretty useful for the user.

Reply:
I don't think it's something we expect to change, so I don't think this is something we should pretend to make variable